### PR TITLE
Schema updated

### PR DESCRIPTION
**Pull request by reenphygeorge**

### CODE REVIEW

File: apps/api/prisma/schema.prisma [hunk 1]
```diff
@@ -23,6 +23,14 @@ model User {
   student             Boolean @default(true)
   organized           Event[]
   participated        TeamMember[]
+
+  organizationId      Organization[]
```
Comment (DarkPhoenix2704) [inline apps/api/prisma/schema.prisma:27]: not id. This is organizations

File: apps/api/prisma/schema.prisma [hunk 2]
```diff
@@ -23,6 +23,14 @@ model User {
...
+}
+
+model Organization {
+    id                      String          @id @unique @db.Uuid @default(uuid())
+    memberId                User[]
```
Comment (DarkPhoenix2704) [inline apps/api/prisma/schema.prisma:32]: Should be members

File: apps/api/prisma/schema.prisma [hunk 3]
```diff
@@ -23,6 +23,14 @@ model User {
...
+    eventId                 Event[]
```
Comment (DarkPhoenix2704) [inline apps/api/prisma/schema.prisma:33]: events

File: apps/api/prisma/schema.prisma [hunk 4]
```diff
@@ -87,6 +99,11 @@ model Tag{
   events    Event[]
 }
 
+model Sponsor {
+  id        String @id @default(dbgenerated("gen_random_uuid()::TEXT"))
+  event     Event[]
```
Comment (DarkPhoenix2704) [inline apps/api/prisma/schema.prisma:104]: Sponsors table should be mapped One-to-Many to event table

**Review by DarkPhoenix2704 — CHANGES_REQUESTED** [review]

**Comment by sreehari2003** [comment]
> _ No description provided. _

@reenphygeorge  i think you should rebase this

**Pull request closed without merging**